src/lib/elements/forms/inputSelect.svelte

```diff
@@ -27,15 +27,29 @@
         error = element.validationMessage;
     };
 
-    $: if (element && required && !value) {
+    const isValid = (value: string | number | boolean) => {
+        if (value === null) return false;
+
+        if (value === undefined) return false;
+
+        if (typeof value !== 'boolean') {
+            if (!value) {
+                return false;
+            }
+        }
+
+        return true;
+    };
```

Comment:
```suggestion
    const isNotEmpty = (value: string | number | boolean) => {
        return typeof value === 'boolean' ? true : !!value
    };
```
I think this works, no?

Reply:
Do you want it to always return `true` if the type is `boolean`? Would there be a case where a boolean value is `null` or `undefined`?

Reply:
`typeof null` returns `object` and `typeof undefined` returns `undefined`, so `typeof value === 'boolean'` only is true when value is either `true` or `false`